met-web/src/components/comments/admin/review/CommentReview.tsx

```diff
@@ -295,12 +327,71 @@ const CommentReview = () => {
                                         onChange={(event) => setOtherReason(event.target.value)}
                                     />
                                     <br />
+                                    <MetParagraph>
+                                        <b>Review Note</b> (this note will be inserted in the email sent to the
+                                        respondent to help them understand what needs to be edited for their comment(s)
+                                        to be approved.)
+                                    </MetParagraph>
+                                    {updatedStaffNote.map((staffNote, index) => {
```

Comment:
Let's create two constant before the return part, const reviewNotes = updatedStaffNote.filter( note => note.type === StaffNoteType[StaffNoteType.Review]. like this then use map to display them here

Reply:
also what does StaffNoteType[StaffNoteType.Review] return?

Reply:
consider something like this:
![image](https://user-images.githubusercontent.com/91914654/211933865-8c92a0f6-6d4a-495a-baaa-d88eb6f732bf.png)

Reply:
addressed it, thanks.